### ISSUE TITLE
Support optional field when JSONEditor is enabled…

### ISSUE THREAD

**Issue by zedrdave** [noirbizarre/flask-restplus#439]
JSONEditor is great, to handle complex PUT/POST schema, however, at the moment, it automatically sends an empty string for fields left empty (even when they are optional/nullable). AFAICT, there is no way to alter this behaviour…

The issue seems to be with swagger-ui, and similar to the one described for this lib: https://stackoverflow.com/questions/48076419/swagger-ui-seems-to-not-handle-optional-keys-with-post-json-body

Basically, we would need to be able to pass:
```
disable_properties:false,
disable_edit_json:false,
```
to the JSON editor being created (as long as some other options, possibly), but these are hardcoded in the current swagger-ui.js…

I know this is mainly a dependency-related issue, but is there any fix we could apply to the template in the meantime?

**Comment by TotempaaltJ** [comment]
Since this is an issue with a dependency and not with Flask RestPlus itself, I don't think this'll be fixed in this repo.

You can override the Swagger UI using the [`documentation()` decorator](http://flask-restplus.readthedocs.io/en/stable/swagger.html#customization). I'd recommend you have a look at [the default implementation](https://github.com/noirbizarre/flask-restplus/blob/96b6530fdc369b92005b470135b609d31e4ccd14/flask_restplus/apidoc.py#L33) for that to start from.
